ui/js/components/perf/compare.js

```diff
@@ -108,3 +108,38 @@ treeherder.component('compareError', {
         newRevision: '<'
     }
 });
+
+treeherder.component('distributionGraph', {
+    template:`
+        <table class = "tooltip-table"> 
+            <tr> 
+                <td class="value-column">{{$ctrl.minValue|abbreviatedNumber : $ctrl.abbreviate}}</td> 
+                <td class="distribution-column"><canvas id="distribution-graph-new" width="190" height="30"></canvas></td>  
+                <td class="value-column">{{$ctrl.maxValue|abbreviatedNumber : $ctrl.abbreviate}}</td> 
+            </tr>
+        </table>`,
+    bindings: {
+        replicates: '<'
+    },
+    controller: [function () {
+        let ctrl = this;
+        let cvs = document.getElementById("distribution-graph-new");
+        let ctx = cvs.getContext("2d");
+        cvs.setAttribute("id", "distribution-graph-current");
+        ctrl.abbreviate = false;
```

Comment:
Hmm, wouldn't we always want to abbreviate? I'm not sure why you made this a parameter.